src/SmartComponents/SystemInventory/SystemInventoryHeader.cy.js

```diff
@@ -29,9 +29,6 @@ describe('Dashboard', () => {
     it('renders correct total with edge hosts', () => {
         cy.get('span[class="insd-c-dashboard__number-data--lg"]').should('have.text', '880');
     });
-    it('the amount of stale systems is correct', () => {
```

Comment:
I removed this test as the code for the next test is essentially the exact same and checks for the same text